src/utils/viewer/viewer.cpp

```diff
@@ -586,20 +596,22 @@ void Viewer::mouseScrollEvent(MouseScrollEvent& event) {
 }
 
 void Viewer::mouseMoveEvent(MouseMoveEvent& event) {
+  if (mousePreviousPosition2D_ == Mn::Vector2i{-1})
+    mousePreviousPosition2D_ = event.position();
+
+  const Mn::Vector2i delta = event.position() - mousePreviousPosition2D_;
+  mousePreviousPosition2D_ = event.position();
```

Comment:
Both SDL and GLFW app provide a convenience [relativePosition()](https://doc.magnum.graphics/magnum/classMagnum_1_1Platform_1_1GlfwApplication_1_1MouseMoveEvent.html#a4954fc274c486ebac749fedc730cf710) that is meant to be used in this case.

The `mousePreviousPosition2D_` member is then not needed anymore.